recipes/pytorchcv/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pytorchcv" %}
+{% set version = "0.0.67" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorchcv-{{ version }}.tar.gz
+  sha256: aa9c64378ff2f0b74624e60ab4529bc4e28042f7bd9fd0e4e43e33f86a442067
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - requests
+    - cython
+    - numpy
+    - cffi
+    - py-opencv
+    - scipy
+    - easydict
+    - matplotlib-base
+    - pillow
+    - pyyaml
+    - visdom
+    - bs4
+    - html5lib
+    - ninja
+    - pytorch
+    - torchvision
+    - jsonpatch
```

Comment:
Where do all those run requirements come from?
Looking at https://github.com/osmr/imgclsmob/blob/f2993d3ce73a2f7ddba05da3891defb08547d504/pytorch/setup.py#L34, there is only `numpy` and `requests`. Obviously `pytorch` is also needed. Are all other packages also required?

Reply:
Thanks for catching this @beenje - I accidentally looked at another project with the same name :(.

I went through the list, and besides `numpy` and `requests` they actually also use `cv2` and `scipy` in some of the models they provide (and heavily use `torchvision`, too). The run deps should now be okay.